lib/capistrano/magento2/defaults.rb

```diff
@@ -60,6 +60,7 @@
 set :magento_deploy_themes, fetch(:magento_deploy_themes, [])
 set :magento_deploy_pearl, fetch(:magento_deploy_pearl, false)
 set :magento_deploy_pearl_stores, fetch(:magento_deploy_pearl_stores, [])
+set :magento_deploy_advanced_bundling, fetch(:magento_deploy_advanced_bundling, false)
```

Comment:
Can you also put this in defaults.rb?

Reply:
This is the defaults.rb file..?

Reply:
Oh yea, oops. Needed more coffee yesterday.